src/Components/DnDLayout/GridTile.tsx

```diff
@@ -169,9 +170,9 @@ const GridTile = ({ widgetType, title, isDragging, setIsDragging, setWidgetAttri
           >
             {title}
           </CardTitle>
-          {link && (
-            <Button className="pf-v5-u-p-0" variant="link" onClick={() => window.open(link.href, '_blank')}>
-              {link.title}
+          {headerLink && (
+            <Button className="widget-header-link pf-v5-u-p-0" variant="link" onClick={() => window.open(headerLink.href, '_blank')}>
```

Comment:
For the future, we should add an `external` flag to the link configuration and have two types of links.

Normal link to enable client-side routing and external link that will open a new tab if the destination is outside of HCC. Unless there is an explicit requirement that all links should open a new tab.